### ISSUE TITLE
Ask confirmation before clearing flow list (key z)

### ISSUE THREAD

**Issue by ntqbit** [mitmproxy/mitmproxy#7413]
#### Problem Description
The `z` keybinding, used to clear flow list, doesn't ask for confirmation, immediately and irreversibly deleting all flows.
It's not impossible to accidentally hit the z key and lose all the flows after a long debugging session.

#### Proposal
Similar to the exit key (`q` or `esc`), it would be nice to add an action confirmation for `z` keybinding to prevent accidental clearing of the flow list.

**Comment by errorxyz** [comment]
I agree with this change and can send a PR if @mhils is okay with this

**Comment by mhils** [comment]
Yes, sounds good to me. 👍

**Comment by Prinzhorn** [comment]
Refs #7282

**Comment by AdityaPatadiya** [comment]
Hello @ntqbit, @mhils, I want to contribute to this issue.
but which files are used to be edited?
I tried but not find the actual code.

**Comment by errorxyz** [comment]
Hey @AdityaPatadiya, sorry for the delay but I'm already in the middle of fixing this. I'll let you know if I'm no longer working on it.

**Comment by AdityaPatadiya** [comment]
Ohh! okay
